src/Codeception/Module/Symfony2.php

```diff
@@ -417,4 +513,14 @@ protected function getInternalDomains()
 
         return array_unique($internalDomains);
     }
+
+    /**
+     * Reboot client's kernel
+     */
```

Comment:
Could you improve a bit documentation of this method? Users should know why and when this should be called.